### ISSUE TITLE
Feature request: Import position of dense_to_sparse and sparse_to_dense.

### ISSUE THREAD

**Issue by 0h-n0** [tensorflow/tensorflow#22927]
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: CentOS 7 
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: No
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.10.1
- **Python version**: 3.6.5
- **Bazel version (if compiling from source)**: No
- **GCC/Compiler version (if compiling from source)**: No
- **CUDA/cuDNN version**: cuda9.0/cuDNN7.0
- **GPU model and memory**: 1080ti
- **Exact command to reproduce**: No

---- 

First of all, I have to appreciate your development. I have a trivial request related to python interface.
Current implementation is the following,

```python
tf.sparse_to_dense
tf.contrib.layers.dense_to_sparse
```

However, This implementation is counterintuitive. I think the following code is more intuitive.

```python
tf.sparse_to_dense
tf.dense_to_sparse
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by 0h-n0** [comment]
I found contrib will be moved to outside repository in tensorflow 2.0 road map. 
 * https://www.tensorflow.org/community/roadmap

I hope tensorflow API will be more user-friendly in tensorflow 2.0. So, this issue wil be solved in  tensorflow 2.0.
